### PR TITLE
layout classes - Yuliia P.

### PR DESCRIPTION
**Pull request by yuliia-p**
Screen Shot 2022-07-01 at 9 00 46 PM

### CODE REVIEW

File: css-layout-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+.fit {
+  display: flex;
+  justify-content: center;
+}
+
+.image {
+  width: 100%;
+  height: 100%;
+}
```
Comment (codingcodymiller) [inline css-layout-classes/styles.css:18]: Where are your rulesets for your `column` classes?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by yuliia-p** [comment]
Screen Shot 2022-07-05 at 5 51 37 PM

File: css-layout-classes/styles.css [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
 }
 
 .row {
   display: flex;
 }
 
-.fit {
-  display: flex;
-  justify-content: center;
+.column-size {
```
Comment (codingcodymiller) [inline css-layout-classes/styles.css:11]: Why isn't this the `column-fourth` rule? This is the class setting the size of the element 🤔

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
You are still missing rules for `column-third`, `column-one-third` (which seems like it would be identical to `column-third` 🤔), and `column-full`.

**Comment by yuliia-p** [comment]
Screen Shot 2022-07-06 at 2 08 06 PM

**Pull request merged**